BUILD.md

```diff
@@ -386,34 +386,28 @@ The DirectML execution provider supports building for both x64 and x86 architect
 ### ARM Compute Library
 See more information on the ACL Execution Provider [here](./docs/execution_providers/ACL-ExecutionProvider.md).
 
-#### Prerequisites
```

Comment:
we should still keep the cross compile instructions that were validated on i.MX8QM but keep it in a separate section?

Reply:
can have separate sections for native compile and cross compile. and specific instructions for validated devices.

Reply:
One of the pre-reqs for cross compiling is the *qemu* package: `sudo apt install -y quemu`